pkg/ui/workspaces/cluster-ui/src/statementDetails/statementDetails.tsx

```diff
@@ -1021,6 +1021,7 @@ export class StatementDetails extends React.Component<
         onSortingChange={this.props.onSortingChange}
         currentScale={this.props.timeScale}
         onChangeTimeScale={this.changeTimeScale}
+        planGists={this.props.statementDetails.statement.stats.plan_gists}
```

Comment:
Is plan_gists always going to exists or do we need to do some optional chaining here?

Reply:
always going to exist